docs/rest/src/concepts.rst

```diff
@@ -83,6 +83,20 @@ understanding the requirements for integration with WOPI clients such as |wac| a
 
         ..  include:: /_fragments/access_token_handling_warning.rst
 
+        **The anonymous access token**
+
+        WOPI operations always require an access token value. However, there are situations where a WOPI host may wish
+        to allow some operations to be anonymously callable. To accomodate such cases, the string ``anonymous``
+        is a reserved access token value in WOPI. Both hosts and clients can use this value in cases where an access
+        token is required, but the host does not actually care to verify the token.
```

Comment:
In the case that the host does not care to verify the token, will a wopi src without an access token still fail with 401?